scripts/creating_variants_summary_table.R

```diff
@@ -0,0 +1,52 @@
+
+
+library(dplyr)
+library(stringr)
+
+get_files <- function ( variants_dir){
+  files <- list.files(path = variants_dir,
+                    pattern = "_variants.csv",
+                    full.names = TRUE,
+                    recursive = FALSE)
+  return (files)
+}
+
+create_summary <- function ( files, output_file ){
+
+  require(dplyr)
+
+  cat(paste("Summarizing", length(files), "variant files."))
+  # TODO: turn this while loop into a vectorized binary function with Reduce and lapply
+
+  i <- 0
+  while ( i != length(files)){
+
+    if ( file.exists (output_file) ) {
+      df1 <- read.table( output_file,
+                         sep = "\t", header = TRUE, colClasses = "character", check.names = FALSE )
+    } else {
```

Comment:
This branch is only taken *once*, isn't it?  Would be much clearer to move this outside of the loop.